### PR TITLE
Fabo/improve action modal

### PR DESCRIPTION
**Pull request by faboweb**
The modal would open and close when data is refetched

### CODE REVIEW

File: src/ActionModal/components/ActionModal.vue [hunk 1]
```diff
@@ -63,16 +58,8 @@
...
-            </span>
-            <TmField
-              id="gas-price"
-              v-model="gasPrice"
-              step="0.000000001"
-              type="number"
-              min="0"
-            />
+            <span class="input-suffix">{{ network.stakingDenom | viewDenom }}</span>
+            <TmField id="gas-price" v-model="gasPrice" step="0.000000001" type="number" min="0" />
```
Comment (Bitcoinera) [inline src/ActionModal/components/ActionModal.vue]: Did you change the linter rules?

**Comment by Bitcoinera** [comment]
I just checked and yes, this PR significantly improves the ActionModal behavior.

So thumb up! :+1:

**Comment by jbibla** [comment]
awesome! 

all the linter changes are concerning but i'll merge anyways.

File: src/ActionModal/components/ActionModal.vue (hunk 1 above)
Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue]: hmm no, we should have a look into standards. maybe my vs code is not working properly

**Pull request merged**